### PR TITLE
Remedy of product

### PR DESCRIPTION
**Pull request by alphaX86**

### CODE REVIEW

**Comment by alphaX86** [comment]
Bro, I made awesome changes and created a signup/login form successfully!

File: src/components/footer.js [hunk 1]
```diff
@@ -51,14 +51,17 @@ const Footer = () => (
...
         <Col xs={12} sm={6} md={4}>
           <h3>About Us</h3>
           <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Repudiandae
-            saepe commodi aut natus. Facere assumenda ea quibusdam adipisci
-            deleniti.
+          We are a group of people who thinks that medication is important for people and even loved ones.
+          We make products which connects customers and dealers to get their medicines on time. 
+          Still confused? Try our site and order your medicines to get right on your doorstep!
           </p>
```
Comment (alphaX86) [inline src/components/footer.js:65]: Change content

File: src/components/footer.js [hunk 2]
```diff
@@ -51,14 +51,17 @@ const Footer = () => (
             <li>
               <FooterLink to="/contact">Contact</FooterLink>
             </li>
+            <li>
+              <FooterLink to="/prod">Products</FooterLink>
+            </li>
```
Comment (alphaX86) [inline src/components/footer.js:56]: Added products link in both header and footer

File: src/components/header.js [hunk 3]
```diff
@@ -42,9 +42,12 @@ const Header = () => (
           <Nav.Link>
             <Link to="/contact">Contact</Link>
           </Nav.Link>
+          <Nav.Link>
+            <Link to="/prod">Products</Link>
+          </Nav.Link>
         </Nav>
         <Nav className="ml-auto mr-3">
-          <Link to="/register">Register/Login</Link>
+          <Link to="/portal">Register/Login</Link>
```
Comment (alphaX86) [inline src/components/header.js:50]: Redirected to portal

File: src/pages/login.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                   Password:
+                   <input type="password" name="pass" required />
+               </label>
+               <input type="submit" value="Login" />
+            </form>
+        </div>
+    </Layout>
+)
+
+export default Login
```
Comment (alphaX86) [inline src/pages/login.js]: Login

File: src/pages/portal.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+            <Link to="/signup">
+            <Button variant="secondary" size="lg" active>
+            Signup
+            </Button>{' '}
+            </Link>
+        </div>
+    </Layout>
+)
+
+export default Portal
```
Comment (alphaX86) [inline src/pages/portal.js:35]: Signup

Comment (alphaX86) [reply src/pages/portal.js:35]: Portal

**Comment by alphaX86** [comment]
If you like this form, merge it!!! 👍

File: src/pages/success.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+const Success = () => (
+    <Layout>
+        <SEO title="SignUp" />
+            <h2>SignUp Successful!</h2>
+            <p>Now you're a customer of VirtuPharm! Welcome aboard!</p>
+            <p>Click to redirect!</p>
+           <Link to="/"><Button variant="success" size="lg">Redirect!</Button></Link>
+    </Layout>
+)
```
Comment (alphaX86) [inline src/pages/success.js:18]: Signup remedy

**Comment by alphaX86** [comment]
Bootstrap icons have been added on forms...(Without interactive ones, just plain)

**Review by BrightonDube — APPROVED** [review]
Great

**Pull request merged**